move.c

```diff
@@ -1,6 +1,8 @@
 #include "move.h"
 
 static int32_t current_cell_start_micrometers;
+/* - Angular acceleration is defined in radians per second squared.*/
```

Comment:
- Remove the `-`?
- Missing space before `*/`

Reply:
41ef97a5c